valgrind.supp

```diff
@@ -0,0 +1,965 @@
+   fun:_ZN2ml4core14printer_detail12CNodePrinterIN5boost17integral_constantIbLb0EEENS0_17CContainerPrinterEE5printINSt3__17complexIdEEEENS9_12basic_stringIcNS9_11char_traitsIcEENS9_9allocatorIcEEEERKT_
+   fun:_ZN2ml4core17CContainerPrinter12printElementINSt3__17complexIdEEEENS3_12basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEERKT_
+   fun:_ZN2ml4core17CContainerPrinter5printINSt3__111__wrap_iterIPKNS3_7complexIdEEEEEENS3_12basic_stringIcNS3_11char_traitsIcEENS3_9allocatorIcEEEET_SG_
+   fun:_ZN2ml4core17CContainerPrinter5printINSt3__16vectorINS3_7complexIdEENS3_9allocatorIS6_EEEEEENS3_12basic_stringIcNS3_11char_traitsIcEENS7_IcEEEERKT_
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Cond
+   fun:_ZNSt3__112basic_stringIcNS_11char_traitsIcEENS_9allocatorIcEEE6appendEPKcm
+   fun:_ZN5boost3log11v2_mt_posix3aux11put_integerIcEEvRNS2_22basic_ostringstreambufIT_NSt3__111char_traitsIS5_EENS6_9allocatorIS5_EEEEjjS5_
+   fun:_ZN5boost3log11v2_mt_posix3aux19date_time_formatterINS2_23decomposed_time_wrapperINS_10posix_time5ptimeEEEcE16format_full_yearERNS8_7contextE
+   fun:_ZNK5boost3log11v2_mt_posix3aux19date_time_formatterINS2_23decomposed_time_wrapperINS_10posix_time5ptimeEEEcEclERNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENSA_9allocatorIcEEEERKS7_
+   fun:_ZNK5boost3log11v2_mt_posix11expressions3aux41date_time_formatter_generator_traits_implINS_10posix_time5ptimeEcE9formatterclERNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENSA_9allocatorIcEEEERKS6_
+   fun:_ZN5boost3log11v2_mt_posix3aux14light_functionIFvRNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENS5_9allocatorIcEEEERKNS_10posix_time5ptimeEEE4implINS1_11expressions3aux41date_time_formatter_generator_traits_implISD_cE9formatterEE11invoke_implEPvSB_SF_
+   fun:_ZNK5boost3log11v2_mt_posix3aux14light_functionIFvRNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENS5_9allocatorIcEEEERKNS_10posix_time5ptimeEEEclESB_SF_
+   fun:_ZNK5boost3log11v2_mt_posix9binder1stIRKNS1_3aux14light_functionIFvRNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENS6_9allocatorIcEEEERKNS_10posix_time5ptimeEEEERNS1_11expressions3aux10stream_refISB_EEEclISE_EEvRKT_
+   fun:_ZN5boost3log11v2_mt_posix15type_dispatcher13callback_base10trampolineINS1_9binder1stIRKNS1_3aux14light_functionIFvRNS1_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENS9_9allocatorIcEEEERKNS_10posix_time5ptimeEEEERNS1_11expressions3aux10stream_refISE_EEEESH_EEvPvRKT0_
+   fun:_ZNK5boost3log11v2_mt_posix15type_dispatcher8callbackINS_10posix_time5ptimeEEclERKS5_
+   fun:_ZN5boost3log11v2_mt_posix10attributes20attribute_value_implINS_10posix_time5ptimeEE8dispatchERNS1_15type_dispatcherE
+   fun:_ZNK5boost3log11v2_mt_posix15attribute_value8dispatchERNS1_15type_dispatcherE
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Cond
+   fun:__lo0bits_D2A
+   fun:__d2b_D2A
+   fun:__dtoa
+   fun:__vfprintf
```

Comment:
There are many suppressions containing a stack with `__vfprintf` and then different functions called from it.  Replacing those with `...` would reduce suppressions.

So this one could be:

```
  ...
  fun:__v2printf
  fun:_vsnprintf
  fun:vsnprintf_l
  fun:snprintf_l
  fun:_ZNKSt3__17num_putIcNS_19ostreambuf_iteratorIcNS_11char_traitsIcEEEEE6do_putES4_RNS_8ios_baseEcd
  fun:_ZNSt3__113basic_ostreamIcNS_11char_traitsIcEEElsEd
  fun:_ZNSt3__1lsIdcNS_11char_traitsIcEEEERNS_13basic_ostreamIT0_T1_EES7_RKNS_7complexIT_EE
```

Then hopefully that means even more suppressions can be consolidated.